src/GovUk.Education.ExploreEducationStatistics.Admin/Services/DataBlockService.cs

```diff
@@ -418,6 +455,7 @@ public class DependentDataBlock
         public string? ContentSectionHeading { get; set; }
         public List<InfographicFileInfo> InfographicFilesInfo { get; set; } = new();
         public bool IsKeyStatistic { get; set; }
+        public bool IsFeaturedTable { get; set; }
```

Comment:
It would be more useful to return some featured table details e.g. the name, so that we can display a more informative message to the end-user rather than the really generic (and unhelpful) one we currently show.

Reply:
I've kept `IsFeaturedTable` and added a new optional `FeaturedTableBasicViewModel` which just contains the name and description to `DependentDataBlock`.